api/proto/common/v2/common_v2.proto

```diff
@@ -36,12 +34,14 @@ message BlobHeader {
 message BlobCertificate {
   // blob_header contains data about the blob.
   BlobHeader blob_header = 1;
-  // relays is the list of relays that are in custody of the blob.
+  // signature over keccak hash of the blob_header that can be verified by blob_header.payment_header.account_id
```

Comment:
Sam suggested a more detailed description for this on my PR for salt. Pasting my changes here for reference but it is outdated for the new structs here
```
// Signature is an ECDSA signature signed by the blob request signer's account ID over the BlobHeader's blobKey,
// which is a keccak hash of the serialized BlobHeader, and used to verify against blob dispersal request's account ID
```